ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -51,8 +51,8 @@ resources:
       cpu: "1"
       memory: 4Gi
 tests:
-- as: alibaba-ipi-fips-p2-f7
-  cron: 48 21 6,14,22,29 * *
+- as: alibaba-ipi-fips-p2-f14
+  cron: 22 18 7,22 * *
```

Comment:
Do we have any customer on alibaba now? As we have aws-ipi-fips, we may consider to decrease it to f28.

Reply:
I am not sure the customer data. But we have the only one alibaba job in this yaml file.

Reply:
Thanks for your clarification. No doubt now.